vite.config.js

```diff
@@ -1,10 +1,19 @@
 import { defineConfig } from 'vite';
 import reactRefresh from '@vitejs/plugin-react-refresh';
+const { PORT = 3001 } = process.env;
```

Comment:
Have you tested if it works without the default value? I think you're missing a `dotenv.config()` directly under the import